plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -827,6 +918,17 @@ def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sock
             if p in kwargs:
                 del kwargs[p]
 
+    # The features work only on PVE 6
+    if PVE_MAJOR_VERSION < 6:
```

Comment:
This branch is not reached as `PVE_MAJOR_VERSION` value is always 3 or 4 currently, resulting in `ciuser`, `cipassword`, `sshkeys` and `ipconfig` being deleted.

Line 1183 should be replaced from this:

    PVE_MAJOR_VERSION = 3 if proxmox_version(proxmox) < LooseVersion('4.0') else 4

To something more like this:

    PVE_MAJOR_VERSION = int(proxmox_version(proxmox).split('.')[0])

Reply:
:see_no_evil: Okay, i've fixed this too.